requirements/dev.in

```diff
@@ -8,22 +8,21 @@ freezegun==1.5.2
 markdown-it-py>=2.2.0
 pipdeptree==2.26.1
 py==1.11.0
-Pygments>=2.15.0  # to bring it up to a secure version
```

Comment:
This has been long resolved to 2.19+ beyond anything vulnerable so is safe to omit.